<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="txt_com">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>70</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="txt_com1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Puerto COM:</string>
    </property>
   </widget>
   <widget class="QLabel" name="txt_estado">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>160</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>ESTADO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_accion">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>CONECTAR</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lista_datos">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>151</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="txt_estado_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>DATOS:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_control">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>270</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>PRENDER</string>
    </property>
   </widget>
   <widget class="QLabel" name="txt_estado_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>230</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>LED:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lista_datos2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>380</y>
      <width>151</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="lista_datos3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>380</y>
      <width>151</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="lista_datos4">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>380</y>
      <width>151</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../3D Objects/facultad_ingenieria_tampico.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../3D Objects/UAT.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>500</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../3D Objects/Castor.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
